packages/repack/src/commands/rspack/start.ts

```diff
@@ -106,12 +110,17 @@ export async function start(
       if (args.interactive) {
         setupInteractions(
           {
-            onReload: () => {
+            onReload() {
               ctx.broadcastToMessageClients({ method: 'reload' });
             },
-            onOpenDevMenu: () => {
+            onOpenDevMenu() {
               ctx.broadcastToMessageClients({ method: 'devMenu' });
             },
+            onOpenDevTools() {
+              void fetch(`${serverURL}/open-debugger`, {
+                method: 'POST',
+              });
```

Comment:
I think we should call this through `ctx` which comes from dev server, no need to call this over the network when we have access to dev server in the same process